GeoFire/GeoFire.h

```diff
@@ -128,4 +128,11 @@ withCompletionBlock:(GFCompletionBlock)block;
  */
 - (GFRegionQuery *)queryWithRegion:(MKCoordinateRegion)region;
 
+
+// private method
+- (FIRDatabaseReference *)firebaseRefForLocationKey:(NSString *)key;
+
++ (CLLocation *)locationFromValue:(id)dict;
+// 这个方法好像没有实现+ (NSDictionary *)dictFromLocation:(CLLocation *)location; 
```

Comment:
seem like you don't implement + (NSDictionary *)dictFromLocation:(CLLocation *)location